--- a/feeds/linkit/mtk-linkit/Makefile
+++ b/feeds/linkit/mtk-linkit/Makefile
@@ -13,10 +13,11 @@
   HIDDEN:=1
   DEPENDS:=@TARGET_ramips_mt7688_LinkIt7688 \
   	+gdbserver +curl +strace +coreutils +coreutils-stty \
-  	+avahi-nodbus-daemon +mountd +mjpg-streamer \
+  	+avahi-dbus-daemon +libavahi-compat-libdnssd +mountd +mjpg-streamer \
 	+uhttpd +rpcd +rpcd-mod-iwinfo +git +git-http +samba36-server \
 	+python +python-pyserial +python-pip +hidapi \
-	+libmraa +libupm +node +node-hid +node-cylon-firmata +yunbridge \
+	+libmraa +libupm +node +node-node-hid +node-cylon-firmata +node-serialport +yunbridge \
+	+node-cylon +node-cylon-gpio +node-cylon-i2c +node-firmata +node-arduino-firmata +node-homebridge \
 	+luci +luci-theme-openwrt +luci-app-mjpg-streamer +luci-app-samba +luci-lib-json \
 	+rpcd-mod-rpcsys +cgi-io +avrdude +spi-tools +uboot-envtools \
 	+kmod-fs-vfat +kmod-fs-exfat +kmod-i2c-core +kmod-i2c-ralink \
